NAMESPACE

```diff
@@ -26,12 +26,12 @@ S3method(visr,attrition)
 S3method(visr,default)
 S3method(visr,survfit)
 export("%>%")
+export(.align_plots)
```

Comment:
keep exported as this function is very useful for other graphs and can be used out of the box

Reply:
It's still exported, no? I just gave it the helper function notation

Reply:
I want to keep it as a utility function from visR as I believe it has added value (so without the . in front)